routing/car_router.cpp

```diff
@@ -470,23 +470,14 @@ CarRouter::ResultCode CarRouter::CalculateRoute(m2::PointD const & startPoint,
   if (startMapping->GetMwmId() == targetMapping->GetMwmId())
   {
     LOG(LINFO, ("Single mwm routing case"));
+    if (!FindRouteMSMT(startTask, m_cachedTargets, delegate, startMapping, route))
+      return RouteNotFound;
```

Comment:
Done. Now we make a cross mwm route in case if there's no error for single mwm case and if a single mwm route is not found.